onnx/version_converter/adapters/upsample_9_8.h

```diff
@@ -0,0 +1,84 @@
+// Adapter for Upsample in default domain from version 9 to 8
+
+#pragma once
+
+#include "onnx/version_converter/adapters/adapter.h"
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+struct Upsample_9_8 final : public Adapter {
+  explicit Upsample_9_8()
+    : Adapter("Upsample", OpSetID(9), OpSetID(8)) {
+    }
+
+  void adapt_upsample_9_8(std::shared_ptr<Graph> graph, Node* node) const {
+      const ArrayRef<Value*>& inputs = node->inputs();
+      const std::vector<Tensor>& initializers = graph->initializers();
+      std::string scale_input_name = node->inputs()[1]->uniqueName();
+    
+      for(size_t i = 0; i < initializers.size(); i++)
+      {
+          if(initializers[i].name() == inputs[1]->uniqueName())
+          {
+            std::vector<float> value = initializers[i].floats();
+            if (initializers[i].is_raw_data()){
+              const std::string& bytes = initializers[i].raw();
+              value.insert(
+                  value.end(),
+                  reinterpret_cast<const float*>(bytes.c_str()),
+                  reinterpret_cast<const float*>(bytes.c_str() + bytes.size()));
+            }            
+            std::vector<double> d_values;
```

Comment:
Wold it be possible to use float here instead of double given that spec takes scales as floats?

Reply:
It would not. fs is using double in IR.